Userland/Libraries/LibDeviceTree/FlattenedDeviceTree.cpp

```diff
@@ -113,11 +113,9 @@ static ErrorOr<ReadonlyBytes> slow_get_property_raw(StringView name, FlattenedDe
     // Name is a path like /path/to/node/property
     Vector<StringView, 16> path;
     TRY(name.for_each_split_view('/', SplitBehavior::Nothing, [&path](StringView view) -> ErrorOr<void> {
-        if (path.size() == path.capacity()) {
+        if (path.size() == path.capacity())
             return Error::from_errno(ENAMETOOLONG);
-        }
-        MUST(path.try_append(view));
-        return {};
+        return path.try_append(view);
```

Comment:
This MUST will always succeed because we cap the Vector at its inline capacity of 16 through the check above

Reply:
(the point of this API is to be allocation-free so we can use it before initializing kmalloc)

Reply:
What would maybe be clearer with `unchecked_append()`, still this won't harm things any more than a `MUST()` :)